web/turkey/survey/static/survey/js/auditors/focus_changes.js

```diff
@@ -0,0 +1,30 @@
+var AuditorFocusChanges = {
+    start_date: new Date(),
+    hidden: null,
+    focus_changes: [],
+    log_focus_changes: function (e) {
+        if(document[this.hidden]) {
+            var focus_change_time = (new Date()).getTime();
+            focus_changes.push(focus_change_time - start_date.getTime());
```

Comment:
See other PR #14 . Auditor parent code requires objects with key/value pairing matching field names and types.